vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -33,32 +36,23 @@
     become: yes
     become_user: root
 
-  - name: Rebooted node status
-    delegate_to: "{{ CLUSTER_APIENDPOINT_IP }}"
-    vars:
-      ansible_user: "{{ IMAGE_USERNAME }}"
-      ansible_ssh_private_key_file: "{{ SSH_PRIVATE_KEY }}"
-      ansible_python_interpreter: /usr/bin/python3
-    become: yes
-    become_user: "{{ IMAGE_USERNAME }}"
-    block:
-      - name: Wait until rebooted k8s node "{{ WORKER_NODE }}" becomes NotReady
-        shell: kubectl get nodes | grep -w NotReady | wc -l
-        environment:
-          KUBECONFIG: "/home/{{ IMAGE_USERNAME }}/.kube/config"
-        retries: 150
-        delay: 3
-        register: not_ready_nodes
-        until: not_ready_nodes.stdout == "1"
-
-      - name: Wait until rebooted k8s node becomes Ready again
-        shell: kubectl get nodes | grep -w Ready | wc -l
-        environment:
-          KUBECONFIG: "/home/{{ IMAGE_USERNAME }}/.kube/config"
-        retries: 150
-        delay: 3
-        register: ready_nodes
-        until: ready_nodes.stdout == NUMBER_OF_BMH
+  - name: Wait until rebooted k8s node "{{ WORKER_NODE }}" becomes NotReady
+    shell: kubectl get nodes | grep -w NotReady | wc -l
+    environment:
+      KUBECONFIG: "/tmp/target.yaml"
```

Comment:
it could also be --kubeconfig in the kubectl command line. I'm fine with both, they are equally clear

Reply:
Sure, done.